--- android_icu4j/src/main/java/android/icu/text/DateFormatSymbols.java	2018-06-13 19:00:05.684113408 +0100
+++ android_icu4j/src/main/java/android/icu/text/DateFormatSymbols.java	2018-06-13 18:58:50.604424107 +0100
@@ -81,7 +81,6 @@
  *
  * @see          DateFormat
  * @see          SimpleDateFormat
- * @see          android.icu.util.SimpleTimeZone
  * @author       Chen-Lieh Huang
  */
 public class DateFormatSymbols implements Serializable, Cloneable {
